game-app/game-headed/src/test/java/games/strategy/engine/auto/update/EngineVersionCheckTest.java

```diff
@@ -3,14 +3,20 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 
-import games.strategy.triplea.settings.AbstractClientSettingTestCase;
 import games.strategy.triplea.settings.ClientSetting;
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.sonatype.goodies.prefs.memory.MemoryPreferences;
 
-final class EngineVersionCheckTest extends AbstractClientSettingTestCase {
+final class EngineVersionCheckTest {
+
+  @BeforeEach
+  void setUp() {
+    ClientSetting.setPreferences(new MemoryPreferences());
+  }
```

Comment:
Had to change this because AbstractClientSettingTestCase is not accessible from game-headed tests